crates/ra_analysis/src/imp.rs

```diff
@@ -3,31 +3,32 @@ use std::{
     sync::Arc,
 };
 
-use ra_editor::{self, find_node_at_offset, FileSymbol, LineIndex, LocalEdit, Severity};
-use ra_syntax::{
-    ast::{self, ArgListOwner, Expr, NameOwner, FnDef},
-    algo::find_covering_node,
-    AstNode, SourceFileNode,
-    SyntaxKind::*,
-    SyntaxNodeRef, TextRange, TextUnit,
-};
-use ra_db::{FilesDatabase, SourceRoot, SourceRootId, SyntaxDatabase};
 use rayon::prelude::*;
 use salsa::{Database, ParallelDatabase};
+
 use hir::{
     self,
-    source_binder,
     FnSignatureInfo,
     Problem,
+    source_binder,
+};
+use ra_db::{FilesDatabase, SourceRoot, SourceRootId, SyntaxDatabase};
+use ra_editor::{self, FileSymbol, find_node_at_offset, LineIndex, LocalEdit, Severity};
+use ra_syntax::{
```

Comment:
seems intellij-rust reorder the use statements. Should i revert it?

Reply:
No, that’s fine